service/k8sclient/kubectl/kubectl.go

```diff
@@ -38,8 +38,9 @@ import (
 )
 
 const (
-	// FIXME: change to a real default kubectl.
-	defaultKubectl = "minikube kubectl --"
+	kubeconfigFileName      = "kubeconfig.json"
```

Comment:
Is it actually used somewhere?

Reply:
Only here https://github.com/percona-platform/dbaas-controller/blob/main/service/k8sclient/kubectl/kubectl.go#L97 and in test https://github.com/percona-platform/dbaas-controller/blob/main/service/k8sclient/kubectl/kubectl_test.go#L54-L61

Do we need to change it to simply `ioutil.TempFile`?

Reply:
Yeah. It doesn't even have to be JSON file – YAML will work, and that's what most people will use

Reply:
Fixed.